Lib/glyphsLib/builder/constants.py

```diff
@@ -60,6 +60,7 @@
 LOCKED_GUIDE_NAME_SUFFIX = " [locked]"
 
 HINTS_LIB_KEY = GLYPHS_PREFIX + "hints"
+SHAPE_SIGNATURE_LIB_KEY = GLYPHLIB_PREFIX + "shapeSignature"
```

Comment:
Wouldn’t it be more future prove to put that into `public.objectLibs`?

Reply:
Yes, proof. 

No, but using that key should be more stable when the file is edited in different tools?